frontend/src/components/pages/your_clubs/ClubList.tsx

```diff
@@ -184,16 +184,24 @@ function ContentWarnings(props) {
   const classes = useStyles();
   const contentWarnings = props.contentWarnings;
 
-  return (
-    <p className={classes.textElement}>
-      <b>Content Warnings:</b>
-      {contentWarnings.map((item, index) => (
-        <>
-          - <b>{item}<br/></b>
-        </>
-      ))}
-    </p>
-  );
+  if (contentWarnings) {
+    return (
+      <p className={classes.textElement}>
+        <b>Content Warnings:</b><br/>
+        {contentWarnings.map((item, index) => (
+          <>
+            - <b>{item}<br/></b>
+          </>
+        ))}
```

Comment:
Consider using `<ul>` and `<li>` here.

Just to go off on an HTML tangent: Both `<b>` and `<br>` are considered bad practice now. The reason is that HTML is meant to specify "what is the content", leaving CSS to specify "how does the content look". This separation is helpful for things like screenreaders and search engine crawlers, as well as just making the code a little cleaner.  In fact, the HTML folks are so committed to this that they retroactively redefined `<b>` from "Bold" (which implies style) to "Bring Attention To" (which just indicates something about the content). They got even sillier with `<i>` ("Idiomatic Text"?) and `<u>` ("Unarticulated Annotation"??). As far as I know they have not yet managed to figure out a semantic meaning for `<br>` so it still means Line Break but I'm sure there's a committee somewhere working on it.

Anyway, this is an unordered list of items and so there are much better semantic tags to use than Bring Attention To and Line Break.